src/requests.ts

```diff
@@ -2,6 +2,14 @@ import Constants from "expo-constants";
 
 const baseUrl = Constants.manifest.extra.uri;
 
+interface Waypoint {
+	route_id: string;
```

Comment:
See the comment in `waypoint-reducer` about `camelCase`

Reply:
See answer to previous comment

Reply:
Fixed